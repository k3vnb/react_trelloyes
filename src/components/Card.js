import React from 'react';


function Card(props){
    return (
        <div className="card_container">
            <h3>
                {props.title}
            </h3>
            <p>{props.content}</p>
        </div>
    );
}



export default Card;