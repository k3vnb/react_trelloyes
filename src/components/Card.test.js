import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer'

describe('Card component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div)
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders in the UI', () => {
        const tree = renderer
            .create(<Card />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})