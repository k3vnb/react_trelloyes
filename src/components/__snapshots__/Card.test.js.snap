// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders in the UI 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    Delete
  </button>
  <h3 />
  <p />
</div>
`;
